src/sp_utils.h

```diff
@@ -28,8 +28,8 @@
 #define HOOK_FUNCTION_BY_REGEXP(regexp, hook_table, new_function) \
   hook_regexp(regexp, SNUFFLEUPAGUS_G(hook_table), new_function)
 
-#define SP_LOG_SIMULATION E_WARNING
-#define SP_LOG_DROP E_ERROR
+#define SP_LOG_SIMULATION 0x100000
```

Comment:
Why change those?

Reply:
Because these error types is a mess for understanding the logs and understanding when on simulation and when dropping the line.
i had to choose a high value that was a single bit within a 32 bit value (int).
i checked them here: https://www.php.net/manual/en/errorfunc.constants.php

by defining these as custom values, i can easily differentiate what is a log and what should be a log related a drop/sim rule.

then internally, in the logger function, we can override the log type with the original values.